presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -219,6 +219,8 @@
     public static final String VERBOSE_RUNTIME_STATS_ENABLED = "verbose_runtime_stats_enabled";
     public static final String STREAMING_FOR_PARTIAL_AGGREGATION_ENABLED = "streaming_for_partial_aggregation_enabled";
     public static final String MAX_STAGE_COUNT_FOR_EAGER_SCHEDULING = "max_stage_count_for_eager_scheduling";
+    public static final String LOWEST_APPROXIMATE_COUNT_DISTINCT_MAX_STANDARD_ERROR = "lowest_approximate_count_distinct_max_standard_error";
+    public static final String LOWEST_APPROXIMATE_SET_MAX_STANDARD_ERROR = "lowest_approximate_set_max_standard_error";
```

Comment:
the DEFAULT_MAX_STANDARD_ERROR for approx_distinct and approx_set seems to have different values, which one should hyperloglog_standard_error_warning_threshold be defaulted to?

Reply:
That is probably fine. Because the underlying data structures are the same. We can use the same small number. The default as suggested by the user is `0.004`.